storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/QueryTest.java

```diff
@@ -143,4 +146,15 @@ public User mapFromCursor(@NonNull Cursor cursor) {
             assertNull(userFromStorage.email());
         }
     }
+
+    @Test
+    public void shouldThrowExceptionIfCursorNullBlocking() {
+        final PreparedGetCursor queryWithNullResult = createQueryWithNullResult();
+        try {
+            queryWithNullResult.executeAsBlocking();
+            fail("StorIOException should be thrown");
+        } catch (StorIOException expected) {
+            // it's okay, cursor was null
```

Comment:
Please add assertion for exception cause type and message, we must be sure that it's really expected exception.